src/content/structured/components/select/code.mdx

```diff
@@ -397,6 +397,133 @@ export const SelectExampleOptionsDisabled = () => (
+  </svg>
+</ic-select>`,
+      long: `{shortCode}
+<script>
+  const select = document.querySelector("ic-select");
+  let option = "Cappuccino";
+  select.options = [
+    { label: "Cappuccino", value: "cappuccino", description: "Coffee frothed up with pressurised steam" },
+    { label: "Americano", value: "americano", description: "Espresso coffee diluted with hot water" },
+    { label: "Mocha", value: "mocha", description: "Coffee with chocolate" },
+  ];
+  select.addEventListener("icChange", function (event) {
+    console.log(event.detail.value);
+  });
+</script>`,
+    },
+  },
+  {
+    technology: "React",
+    snippets: {
+      short: `<IcSelect 
+  label="What is your favourite coffee?"
+  options={options}
+  onIcChange={(event) => console.log(event.detail.value)}
+>
+  <SlottedSVG
+    slot="icon"
+    xmlns="http://www.w3.org/2000/svg"
+    height="24"
+    viewBox="0 0 24 24"
+    width="24"
+    fill="#000000"
+  >
+    <path d="M2,21H20V19H2M20,8H18V5H20M20,3H4V13A4,4 0 0,0 8,17H14A4,4 0 0,0 18,13V10H20A2,2 0 0,0 22,8V5C22,3.89 21.1,3 20,3Z" />
+  </SlottedSVG>
+</IcSelect>`,
+      long: [
+        {
+          language: "Typescript",
+          snippet: `const options = [
+   { label: "Espresso", value: "espresso" },
+   { label: "Double Espresso", value: "doubleespresso" },
+   { label: "Flat White", value: "flatwhite" },
+   { label: "Cappuccino", value: "cappuccino" },
+   { label: "Americano", value: "americano" },
+   { label: "Mocha", value: "mocha" },
+ ];
+ return (
+   {shortCode}
+ )`,
+        },
+        {
+          language: "Javascript",
+          snippet: `const options = [
+   { label: "Espresso", value: "espresso" },
+   { label: "Double Espresso", value: "doubleespresso" },
+   { label: "Flat White", value: "flatwhite" },
+   { label: "Cappuccino", value: "cappuccino" },
+   { label: "Americano", value: "americano" },
+   { label: "Mocha", value: "mocha" },
+ ];
+ return (
+   {shortCode}
+ )`,
+        },
+      ],
+    },
+  },
+];
+
+export const SelectExampleWithIcon = () => (
+  <IcSelect
+    label="What is your favourite coffee?"
+    options={[
+      {
+        label: "Cappuccino",
+        value: "cappuccino",
+        description: "Coffee frothed up with pressurised steam",
+      },
+      {
+        label: "Americano",
+        value: "americano",
+        description: "Espresso coffee diluted with hot water",
+      },
+      {
+        label: "Mocha",
+        value: "mocha",
+        description: "Coffee with chocolate",
+      },
+    ]}
+    onIcChange={(event) => console.log(event.detail.value)}
+  >
+    <svg
+      slot="icon"
+      xmlns="http://www.w3.org/2000/svg"
+      height="24"
+      viewBox="0 0 24 24"
+      width="24"
+      fill="#000000"
+    >
+      <path d="M2,21H20V19H2M20,8H18V5H20M20,3H4V13A4,4 0 0,0 8,17H14A4,4 0 0,0 18,13V10H20A2,2 0 0,0 22,8V5C22,3.89 21.1,3 20,3Z" />
+    </svg>
```

Comment:
```suggestion
    <SlottedSVG
      slot="icon"
      xmlns="http://www.w3.org/2000/svg"
      height="24"
      viewBox="0 0 24 24"
      width="24"
      fill="#000000"
    >
      <path d="M2,21H20V19H2M20,8H18V5H20M20,3H4V13A4,4 0 0,0 8,17H14A4,4 0 0,0 18,13V10H20A2,2 0 0,0 22,8V5C22,3.89 21.1,3 20,3Z" />
    </SlottedSVG>
```

Reply:
Updating to this seems to remove the icon, which is odd as it works using SlottedSVG in the stackblitz example. Wondering if its just a Gatsby quirk

Reply:
That's weird 🤔. Happy to approve in this case